docs/source/developer_resources/Execution_Workflow.rst

```diff
@@ -14,3 +14,7 @@ Simulator - In this case GridLAB-D serves as the simulator.
 
 Hosted Application - Applications can be developed to use the data generated by the simulation and submit feedback and updates to the simulator.  Two examples of this have been developed in RC1, the VoltVar application and a vizualization application
 
+Log Manager - Process Manager recieves a log message. It retrieves the username associated with the message and forwards the message and username to Log Manager. Log Manager writes the message on a file and if store_to_db key is true in log message then log manager calls the data manager to store the log message in the database.
```

Comment:
Perhaps it would be good to use https://www.websequencediagrams.com/ and see if we can have a picture for the sequence.

Reply:
I like the tool. We need to update the current sequence diagram as well. We will need multiple sequence diagrams based on different requests and corresponding managers. I will create separate task for it so it is not blocking Rc2 tagging.

Reply:
![new simulation sequence](https://user-images.githubusercontent.com/6538501/33401979-054dd090-d510-11e7-950c-cd46f7db7d0a.png)
I created a sample one like this for a new simulation request.

Reply:
Awesome...I like it...we should have more of these I think